index.html

```diff
@@ -214,10 +214,10 @@ <h3>
               <p>
 This specification defines how to express proofs of integrity, such as digital
 signatures or proofs of existence, for bounded documents, such as verifiable
-credentials. Concrete serializations will be provided for <a
+credentials. Concrete serializations will be provided based on <a
 href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a> and the <a
 href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>.
-Concrete serializations might be provided for a <a
+Concrete serializations might be provided based on the <a
 href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a>, if an <a
 href="https://identity.foundation/bbs-signature/">IETF BBS+ RFC</a> is published
 in time, as well as VC-JWP, if an <a
```

Comment:
```suggestion
Concrete serializations based on the <a
href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a> might be
provided, if an <a
href="https://identity.foundation/bbs-signature/">IETF BBS+ RFC</a> is published
in time, as well as VC-JWP, if an <a
```

Reply:
The latter sentence might be better broken into two, even if they are worded almost identically.

Reply:
Yeah, I was on the fence about that as well. I kept it as the same sentence to try and convey "these two things are in the same category of "if an IETF RFC appears, we can standardize it."

Reply:
I don't think the categorization of these two things is particularly important, because they're waiting on different RFCs, which are not tied together -- either RFC might be published, or both, or neither.  On that basis, I think it will actually be significantly clearer to split the sentence into two -- one for each serialization which might be provided, each dependent on publication of one RFC.

Reply:
Ok, agree with your logic @TallTed, I will make that change when I get to my next edit cycle.

Reply:
Done in https://github.com/w3c/vc-wg-charter/pull/37/commits/8abd5831d96ebe323ad5fc9c05faa89c62fa48e1.